src/ui/server/test-mocks/mock-exporter-business-nature-of-business.ts

```diff
@@ -1,10 +1,10 @@
 import { ExporterBusiness } from '../../types';
 
-const mockExporterBusiness = {
+const mockExporterBusinessNatureOfBusiness = {
   goodsOrServicesSupplied: 'ABC',
   totalYearsExporting: '2,0',
   totalEmployeesInternational: '1,000',
   totalEmployeesUK: '4,00',
 } as ExporterBusiness;
 
-export default mockExporterBusiness;
+export default mockExporterBusinessNatureOfBusiness;
```

Comment:
Can we align all the test mocks here to be inline with the DB structure/what would get returned from the API? We've been doing that for other areas e.g

```js
const mockApplication = {
  // ...
  policyAndExports: { ... },
  exporterCompany: { ... },
  exporterBusiness: { ... },
};
```

I.e there's no "business turnover or "nature of business" in the structure it's all under `exporterBusiness`

Reply:
I am using these mocks for what the body could be, if I put them all in one, the test would not be representative of the update (turnover would not get natureOfBusiness fields)

Reply:
Where are these mocks used in this manner? I can't see

Beacuse the idea is this mock application is what we'd get from the API i.e

```js
application {
  policyAndExport { ... },
  exporterCompany: { ... },
  exporterBusiness: { ... }
}
```

The nature of business / turnover fields are inside `exporterBusiness` and doesn't the actual api/DB update only  `exporterBusiness`? Bit confused why it's structured differently in the mocks.